README.md

```diff
@@ -82,17 +82,17 @@ cd nessie
 
 Nessie Iceberg's integration is compatible with Iceberg as in the following table:
 
-| Nessie version  | Iceberg version       | Spark version          | Hive version   | Flink version |
-| --------------- | --------------------- | ---------------------- | ---------------|---------------|
-| 0.18.0          | 0.13.0(yet to release)| 3.0.X, 3.1.X           | 2.3.9          | 1.12.1        |
-| 0.9.2           | 0.12.1, 0.12.0        | 3.0.X, 3.1.X           | 2.3.9          | 1.12.1        |
+| Nessie version | Iceberg version       | Spark version          | Hive version   | Flink version |
+|----------------| --------------------- | ---------------------- | ---------------|---------------|
+| 0.19.0         | 0.13.0(yet to release)| 3.0.X, 3.1.X           | 2.3.9          | 1.12.1        |
```

Comment:
nit: we can remove yet to release. jars are released yesterday

Reply:
Interesting, I've looked for a Git tag, but there's none - but it's already in Maven Central